stages/org.osbuild.cpio.out

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python3
+"""
+Assembles the tree into a CPIO archive named `filename`.
+
+Uses the buildhost's `cpio` command, to create an archive
+at `filename` with the contents of the input `tree`. If
+`append` is `true`, it files will be added to an existing
+archive. The default format is `newc" , the "new (SVR4)
+portable format", which is also used by `dracut`(8).
+
+Buildhost commands used: `cpio`
+"""
+
+import os
+import subprocess
+import sys
+
+import osbuild.api
+
+CAPABILITIES = ["CAP_MAC_ADMIN"]
```

Comment:
Can you elaborate what this is used for?

Reply:
In `tar` we need it to read unknown SELinux labels, but grep'ing the `cpio` 2.13 source for SELinux or extended attributes yields nothing. I am dropping this. Thanks!